pytest_fixtures/core/sys.py

```diff
@@ -53,18 +53,10 @@ def puppet_proxy_port_range(session_puppet_enabled_sat):
 
 @pytest.fixture(scope='class')
 def install_cockpit_plugin(class_target_sat):
-    class_target_sat.download_repofile()
+    class_target_sat.create_custom_repos(
+        **settings.repos[f'rhel{class_target_sat.os_version.major}_os']
+    )
```

Comment:
It would have been better if you registered with CDN and enabled OHSNAP_RHEL8_REPOS instead of setting up custom RHEL repos